### ISSUE TITLE
Test Front End Application under /talentmap

### ISSUE THREAD

**Issue by burgwyn** [18F/State-TalentMAP#1021]
- Build the front end to run under a directory
- Create `http.conf` for Apache

**Comment by afrasier** [comment]
I'm fairly certain we could just `cp /var/www/html/ /var/www/html/talentmap/`, right?

**Comment by burgwyn** [comment]
Certainly, but there is some additional config that we should capture.  I’m thinking CSP and any other global headers.

**Comment by burgwyn** [comment]
Handed off a build with updated config.  I want to explore further utilizing webpack before closing.
